themes/Cherry-Red/themeStyle.css

```
@@ -0,0 +1,1225 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Acourd
```

```suggestion
 * SPDX-FileCopyrightText: 2022 Acourd
 * SPDX-FileCopyrightText: 2024 SuperZekes
```

If you made any additional changes, add your name too (don't remove the old though)